week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -11,12 +11,15 @@ We made a start on this for you here:
 let bottle = {
   volume: 0,
   fill: function() {
+    this.volume = 100;
     // calling this function should make you bottles volume = 100;
   },
   drink: function() {
+    this.volume = this.volume - 10;

```

Comment:
Extra tip - check here Assignment operators for a different way to do this ;)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Assignment_Operators#Subtraction_assignment